core/sup_tree_core/cluster_hosts_poller.ex

```diff
@@ -21,7 +21,7 @@ defmodule AntikytheraCore.ClusterHostsPoller do
     end
   end
 
-  def start_link() do
+  def start_link(_args) do
```

Comment:
Let's change `_args` to `[]` to make the incoming arguments explict.

Reply:
fixed in a2bb8aa